src/main/java/com/rocketden/main/controller/v1/BaseRestController.java

```diff
@@ -0,0 +1,8 @@
+package com.rocketden.main.controller.v1;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@RequestMapping("/api/v1")
+public class BaseRestController {
+    // empty
```

Comment:
nit: Remove comment, or edit it to explain how the **BaseRestController** works.

Reply:
Added a brief comment describing the class.